inst/json/metadata_0.1.0.json

```
@@ -81,8 +81,8 @@
       { "id": "education_closures", "label": "Closures" },
       { "id": "education_absences", "label": "Absences" },
       { "id": "life_years", "label": "Life years" },
-      { "id": "life_years_infants", "label": "Infants" },
-      { "id": "life_years_adolescents", "label": "Adolescents" },
+      { "id": "life_years_pre_school", "label": "Pre-school-age children"},
```

Most sources spell 'preschool' without any space or hyphen, which looks neater to me in this context!
```suggestion
      { "id": "life_years_pre_school", "label": "Preschool-age children"},
```

---

How unpleasant, but yes avoids the double hyphen